projects/igniteui-angular/src/lib/grids/grid-public-cell.ts

```diff
@@ -273,7 +288,14 @@ export class IgxGridCell implements CellType {
     }
 
     private isCellInEditMode(): boolean {
-        return this.grid.crudService.cellInEditMode;
+        if (this.grid.crudService.cellInEditMode) {
+            const cellInEditMode = this.grid.crudService.cell.id;
+            const isCurrentCell = cellInEditMode.rowID === this.id.rowID &&
+                cellInEditMode.rowIndex === this.id.rowIndex &&
+                cellInEditMode.columnID === this.searchColumnIndexByName(cellInEditMode.columnID, this._columnField);
```

Comment:
I like this approach with introducing a method to look for a column by index(check visibleIndex too). But to make it more generic, it is the `get column` method to be fixed, so that correctresult is alwways returned, rather than fixed only for this case in the isCellinEditMode method